projects/testing-library/tests/rerender.spec.ts

```diff
@@ -48,6 +49,41 @@ test('rerenders the component with updated inputs', async () => {
 });
 
 test('rerenders the component with updated props and resets other props', async () => {
```

Comment:
[optional] This little change helps me to faster understand the difference to next test.

```suggestion
test('rerenders the component with updated inputs and resets other props', async () => {
```